clusters/build-clusters/common/monitoring_viewer.yaml

```diff
@@ -9,12 +9,4 @@ roleRef:
   name: cluster-monitoring-view
 subjects:
 - kind: Group
-  name: cluster-monitoring-viewer
----
-apiVersion: user.openshift.io/v1
-kind: Group
-metadata:
-  name: cluster-monitoring-viewer
-users:
-- YuviGold
-- wking
```

Comment:
wking is added into the rover group.

I cannot see YuviGold in github openshift org: https://github.com/orgs/openshift/people
Either in rover.